### PR TITLE
testing pull requests

### PR DESCRIPTION
**Pull request by zongchen-yang**
testing pull req

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const path = require('path');
 const colors = require('colors');
 const axios = require('axios');
+// adding morgan
```
Comment (spencerasava) [inline server/index.js:5]: GOOD JOB!!!

Comment (ebbarker) [reply server/index.js:5]: lgtm

**Review by out-spaced — APPROVED** [review]
I approve

**Review by spencerasava — APPROVED** [review]
looks good ready to merge

**Pull request merged**